conf/logback.xml

```diff
@@ -15,7 +15,7 @@
   <logger name="play" level="INFO" />
   <logger name="application" level="DEBUG" />
 
-  <root level="ERROR">
+  <root level="INFO">
     <appender-ref ref="STDOUT" />
```

Comment:
Thanks, good advice. Forgot the console logs appended to a file automatically with play. For now I've reset this back to `ERROR`. Can add a rolling file appender later.